### DISCUSSION TITLE
could you build Organic Maps (or include the APK) in an AOSP based custom rom?

### DISCUSSION THREAD

**Discussion by dot166** [organicmaps/organicmaps#6604] [Q&A] [answered]
could it work???

**Comment by hb0nd** [comment]
it's completely possible, but that's up to the ROM developers, not OM :)

**Reply by dot166** [reply]
But how would you do it? Because I am trying to make a ROM (FOR PERSONAL USE) for my phone and I would like to include OM but I cannot figure out how to do it.

**Reply by hb0nd** [reply, marked as answer]
out of scope for this repository i'm afraid, but https://www.androidauthority.com/build-custom-android-rom-720453/ might be a good start

**Reply by dot166** [reply]
Oh. Thanks anyways

**Comment by hb0nd** [comment]
> But how would you do it? Because I am trying to make a ROM (FOR PERSONAL USE) for my phone and I would like to include OM but I cannot figure out how to do it.
